app/views/about/index.html.haml

```diff
@@ -25,9 +25,15 @@
         about Hack Week.
     %p.lead
       What are you going to hack on?
+      = image_tag 'monsterhacks_monsters_only.png', class: 'center-block', alt: 'Hack Week 17'
   .col-md-5
     %div
-    = image_tag 'hackweek_monsterhacks.png', class: 'img-responsive', alt: 'Hack Week 17'
+      .h3 Welcome Video from Ralf Flaxa
+      .p  President of Engineering at SUSE
+      .embed-responsive.embed-responsive-16by9#intro
+        %video{:width => 320, :height => 240, :controls => true}
+          %source{:src => "#{video_path('Hackweek_Welcome_by_Ralf.mp4')}", :type => 'video/mp4; codecs="avc1.42E01E, mp4a.40.2"'}
```

Comment:
This could be `%source{ src: "#{video_path('Hackweek_Welcome_by_Ralf.mp4')}", type: 'video/mp4; codecs="avc1.42E01E, mp4a.40.2"' }`